### ISSUE TITLE
Troubles with token

### ISSUE THREAD

**Issue by benvindo** [patriz/falcon-rest-api#4]
Hi,

The Token doesn't expire and it continues to function even after deleting the user.

**Comment by luisliz** [comment]
Yeah, noticed this and gives access to the whole API. Is this intended to work this way?
